streamlit_ace/__init__.py

```diff
@@ -113,13 +117,16 @@ def st_ace(
         annotations=annotations,
         markers=markers,
         name=key or "ace-editor",
-        key=key or "ace-editor"
+        key=key,
```

Comment:
`None` is a valid key, so it shouldn't be overridden. (This will cause an error if you add multiple st_ace() components to the same page, and just use the default `key`)